Framework/CurveFitting/src/ComplexMatrix.cpp

```diff
@@ -301,8 +301,6 @@ void ComplexMatrix::eigenSystemHermitian(GSLVector &eigenValues,
   eigenValues.resize(n);
   eigenVectors.resize(n, n);
   auto workspace = gsl_eigen_hermv_alloc(n);
-  // RT: I think there is a bug in this function. It returns garbage in
-  // eigenvectors in a repeated call.
```

Comment:
has this bug been fixed?

Reply:
The bug wasn't in the gsl. And it has been fixed.